src/callbacks.c

```diff
@@ -274,6 +274,9 @@ gboolean on_buttonpress (GtkWidget *win,
 			      devdata->cur_context->minwidth) +
 		    devdata->cur_context->minwidth);
 
+  if(data->maxwidth > devdata->cur_context->maxwidth)
```

Comment:
This sets the width to 0 for the default tools and for tools not having `maxsize` config set. I.e. makes them invisible ;-) You should probably default to ` G_MAXUINT `.